vms/platformvm/block/executor/verifier_test.go

```diff
@@ -29,7 +29,7 @@ import (
 	"github.com/ava-labs/avalanchego/vms/platformvm/txs/mempool"
 )
 
-func TestVerifierVisitProposalBlock(t *testing.T) {
+func TestApricotVerifierVisitProposalBlock(t *testing.T) {
```

Comment:
As usual this tests are not well maintained.
Introducing the following convention:
The first word after test is the fork that the test targets. We use LatestFork it the test is meant to be carried out across forks, so to remember to update the test.